server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -929,6 +930,7 @@ public void execute(final String[] args) {
     Opts opts = new Opts();
     opts.parseArgs("accumulo init", args);
     SiteConfiguration siteConfig = new SiteConfiguration();
+    ConfigSanityCheck.validate(siteConfig);
```

Comment:
Maybe we should just validate the site config when it is constructed, instead of doing this as a separate task?

Reply:
I have been in the 1.9 branch recently and I remember seeing the following.  Not sure if this was carried forward in 2.0 refactoring to remove singleton.

https://github.com/apache/accumulo/blob/rel/1.9.2/core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java#L85